Composer/packages/lib/indexers/src/luIndexer.ts

```diff
@@ -1,43 +1,78 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import ludown from 'ludown';
+import { parser } from '@bfcomposer/bf-lu/lib/parser';
+import { Position, Range, Diagnostic, DiagnosticSeverity } from 'botbuilder-lg';
 
-import { FileInfo, LuFile, LuDiagnostic } from './type';
+import { FileInfo, LuFile } from './type';
 import { getBaseName } from './utils/help';
 
+function parseDiagnostics(text: string) {
```

Comment:
This should have a return type.

Reply:
Also, please add a unit test for this.

Reply:
We should not parse a diagnostic string, we should directly get a `Diagnostic` object from lu-parser, instead of a plain string and then paring that.  

If that's not possible, that's an issue of lu-parser. we should bring up there.